src/main/java/com/ea/restaurant/grpc/GrpcMongoOrderStatusHistoryServiceImpl.java

```diff
@@ -40,7 +40,7 @@ public void insertMongoOrderStatusHistoriesFromPythonEtl(
               .build());
     } catch (Exception e) {
       responseObserver.onError(
-          Status.INTERNAL.withDescription(e.getMessage()).withCause(e).asRuntimeException());
+          Status.PERMISSION_DENIED.withDescription(e.getMessage()).withCause(e).asRuntimeException());
```

Comment:
This should be an internal error. Why are you sending a `PERMISSION_DENIED` error if something happens within `insertMongoOrderStatusHistoriesFromPythonEtl`?

I don't think any of this operations will generate a `PERMISSION_DENIED` exception.

Reply:
I was a mistake. i fixed.